mgr.go

```diff
@@ -627,6 +648,37 @@ func (mgr *SysboxMgr) unregister(id string) error {
 		info.shiftfsMarks = []configs.ShiftfsMount{}
 	}
 
+	// If the rootfs is ID-mapped and on overlayfs, then chown the upper dir from
+	// [userns-host-ID -> 0] when the container stops, as otherwise the upper dir
+	// will have the host uid:gid assigned to the container when it restarts
+	// (because overlayfs does not support ID-mapping the upper dir). Skip if the
```

Comment:
"... support Id-mapping *in* the upper dir"

Reply:
It's fine to say "ID-mapping the upper dir" because the ID-map is applied to an entire dir.

Reply:
Gotcha.